pkg/_pub_shared/lib/data/package_api.dart

```diff
@@ -196,6 +196,10 @@ class VersionInfo {
   @JsonKey(name: 'archive_url')
   final String? archiveUrl;
 
+  /// The SHA-256 hash of the canonical archive.
+  @JsonKey(name: 'archive_sha256')
```

Comment:
I don't know why we would ever get a null here, but if we have I think it would be better not to include the attribute in the json.

```suggestion
  @JsonKey(name: 'archive_sha256', includeIfNull: false)
```